cityofaustin/atd-moped
# replaces moped entities with 0 id with null values

## Associated issues
fixes https://github.com/cityofaustin/atd-data-tech/issues/10594

## Testing
**URL to test:** 

[10594_remove_null_record_moped_entity--atd-moped-main.netlify.app/moped/](https://10594_remove_null_record_moped_entity--atd-moped-main.netlify.app/moped/)

**Steps to test:**
- go to a project summary page
- if there is no project lead, add one and save it
- clear the project lead and save it
- the update should be successful and you should see a "-" in the project lead field
- repeat the above steps for project sponsor

---
#### Ship list
